src/xdocs/config_whitespace.xml

```diff
@@ -2976,6 +2980,22 @@ public void myTest() {
 
     try {} catch (Exception e){} // OK
     try{} catch (Exception e){} // violation ''try' is not followed by whitespace'
+
+    try {
+        Integer.parseInt(args[0]);
+    } catch (NumberFormatException e) { // OK
```

Comment:
Make examples single line.
Look at https://github.com/checkstyle/checkstyle/pull/11496 for example

Reply:
Making the same example single line would be fine?

Reply:
Yes, please make the same example in the single line.
Like: 
`try { nteger.parseInt(args[0]); } catch (NumberFormatException e) { // OK`

I would suggest you remove `try `content and use `Exception `in the catch block to avoid a 100+ line limit.
Eg: 
`try { } catch (Exception e) { // OK`